src/Components/StudentsList/Pagination.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+
+function Pagination() {
+  const [page, setPage] = useState(1);
+  const [students, setStudents] = useState([]);
+
+  useEffect(() => {
+    //  (fetch from BE (get students page - pageNo?) and setStudents)()
+  }, [page]);
+
+  // const content = students.map() over the student's data - AvailableStudentItem ? ;
+
+  const nextPage = () => setPage((prev) => prev + 1);
+
+  const prevPage = () => setPage((prev) => prev - 1);
+
+  return (
+    <>
+      Ilosc elementow
+      <nav>10</nav>
+      10 z 90
+      <nav>
+        <button onClick={prevPage} disabled={page === 1}>
```

Comment:
It woud be worth to use button component (Components/Common) if suitable, if not you could create new one.